### ISSUE TITLE
Category, Fields Published and Hidden - Is there a difference?

### ISSUE THREAD

**Issue by AlexanderSupp** [JoomGalleryfriends/JG4-dev#155]
Category - Tab Category - Field Published Yes/No 
Category - Tab Options - Field Hidden Yes/No

**Comment by MrMusic** [comment]
See the description:
_With hiding this category it won't be displayed in gallery or category view anymore, but it will still be available through direct linking.
So please note that "hiding" this is not a protection for it. If you don't want it to be available please unpublish it instead.
By hiding categories it is possible to create separate areas in the gallery, for example. For that please create a menu item for the hidden categories._ 

In other words:
if a category is 'unpublished' it (and subcategories and images contained therein) cannot be displayed in the frontend, i.e. as if it did not exist at all.
if a category is 'hidden' it becomes invisible in the category view. It (and subcategories and images contained in it) can be accessed through a direct link (menu link).

**Comment by AlexanderSupp** [comment]
Thanks, for your quick response.
